ntbs-build/openshift-pipelines/tasks/parse-gradle-results.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: parse-gradle-results
+  namespace: ntbs-build
+spec:
+  params:
+    - name: gradle_directory
+      type: string
+  steps:
+    - image: >-
+        image-registry.openshift-image-registry.svc:5000/ntbs-build/ntbs-image-stream:node-alpine
+      name: parse-results
+      script: |
```

Comment:
is this missing "-" at the start of name and script? (I think this actually might be git not getting the colours quite right)

Reply:
They syntax here is correct - the dash signifies the start of a new item in the list. I agree that it's a bit weird, but it's essentially a short-hand. i.e. rather than something like:
```
list:
  - item:
    - propertyA: 1
    - propertyB: true
  - item:
    - propertyA: 3
    - propertyB: false
```
we just do
```
list:
  - propertyA: 1
    propertyB: true
  - propertyA: 3
    propertyB: false
```

Reply:
In this case, there's only one item in the list of steps

Reply:
Ahh okay